miniwin/miniwin/src/miniwin_ddraw.cpp

```diff
@@ -82,7 +82,7 @@ HRESULT DirectDrawImpl::CreateSurface(
 			if ((lpDDSurfaceDesc->dwFlags & DDSD_ZBUFFERBITDEPTH) != DDSD_ZBUFFERBITDEPTH) {
 				return DDERR_INVALIDPARAMS;
 			}
-			SDL_Log("Todo: Set %dbit Z-Buffer", lpDDSurfaceDesc->dwZBufferBitDepth);
+			// SDL doesn't expose the z-buffer and handles it automatically; return a dummy surface.
 			*lplpDDSurface = static_cast<IDirectDrawSurface*>(new DirectDrawSurfaceImpl);
```

Comment:
Won't we have to create a depth buffer, when we're going to render (multiple) 3d objects.

Reply:
Oh I think I miss understood some docs, I thought it was done automatically by SDL. My headspace is still very much around fixed function graphics :D

Reply:
It's a buffer + stencil test: https://www.youtube.com/watch?v=NXY00vKi3iA